### PR TITLE
added contributeToTentative to flash scripts

### LINKED ISSUE #3018: Flash script: "contributeToTentative"
create flash script to add REP to current tentative winning outcome.

### PR DESCRIPTION
**Pull request by bthaile**
fixes https://github.com/AugurProject/augur/issues/3018

only added  contributeToTentative to flash script. Will add fork in a different PR.

fixed some copy paste parameter descriptions.
added some pass throughs in contract-api.

### CODE REVIEW

File: packages/augur-tools/src/flash/scripts.ts [hunk 1]
```diff
@@ -444,14 +468,82 @@ export function addScripts(flash: FlashSession) {
...
+        abbr: 'o',
+        description:
+          'outcome of the market, non scalar markets 0,1,3,... for scalar markets use price',
+        required: true,
+      },
+      {
+        name: 'amount',
+        abbr: 'a',
+        description: 'amount of REP to dispute with, use display value',
+        required: false,
```
Comment (robert-e-davidson3) [inline packages/augur-tools/src/flash/scripts.ts]: I think this has to be `true` due to line #510, where you do `new BigNumber(amount)`. A missing `amount` is going to be an empty string (I think?) not `'0'`.

Comment (bthaile) [reply packages/augur-tools/src/flash/scripts.ts]: ok, prob copy paste error, will check it out

**Review by robert-e-davidson3 — CHANGES_REQUESTED** [review]
Flash script's `amount` parameter needs to be `required: true`.

File: packages/augur-tools/src/flash/scripts.ts (hunk 1 above)
Comment (bthaile) [reply packages/augur-tools/src/flash/scripts.ts]: good catch, pushed change

**Review by robert-e-davidson3 — APPROVED** [review]
LGTM!

**Pull request merged**